luigi/worker.py

```diff
@@ -182,7 +182,14 @@ def run(self):
             # checking completeness of self.task so outputs of dependencies are
             # irrelevant.
             if self.check_unfulfilled_deps and not _is_external(self.task):
-                missing = [dep.task_id for dep in self.task.deps() if not self.check_complete(dep)]
+                missing = []
+                for dep in self.task.deps():
+                    if not self.check_complete(dep):
+                        nonexistent_outputs = [output for output in dep.output() if not output.exists()]
+                        if nonexistent_outputs:
+                            missing.append('%s (%s)' % (dep.task_id, ', '.join(map(str, nonexistent_outputs))))
```

Comment:
would really prefer `str.format()` or f-strings for readability

Reply:
```suggestion
                            missing.append(f'{dep.task_id} ({", ".join(map(str, nonexistent_outputs))})')
```

Ok, I was only following the existing style